tests/end-to-end/backup-restore-test/restore_cluster_backup_test.go

```diff
@@ -0,0 +1,88 @@
+package restore
+
+import (
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	. "github.com/kyma-project/kyma/tests/end-to-end/backup-restore-test/backupe2e"
+	"github.com/kyma-project/kyma/tests/end-to-end/backup-restore-test/utils"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+var testUUID = uuid.New()
+var backupName = "test-" + testUUID.String()
+
+type e2eTest struct {
+	backupTest BackupTest
+	namespace  string
+	testUUID   string
+}
+
+func TestBackupAndRestoreCluster(t *testing.T) {
+	myFunctionTest, err := NewFunctionTest()
+
+	if err != nil {
+		t.Fatalf("%v", err)
+	}
+	backupTests := []BackupTest{myFunctionTest}
+
+	e2eTests := make([]e2eTest, len(backupTests))
+
+	for idx, backupTest := range backupTests {
+		testUUID := uuid.New()
+		e2eTests[idx] = e2eTest{
+			backupTest: backupTest,
+			namespace:  "test-" + testUUID.String(),
```

Comment:
can there be a possibility if more than one namespaces are required by the tests.

Reply:
sure, but then its up to them to create them